### PR TITLE
Cinder Role: removed icehouse and havana legacy workings, changed structure to mai…

### PR DESCRIPTION
**Pull request by jdextercsc**
…n linking to a release specific included play

### CODE REVIEW

File: roles/cinder/tasks/main.yml [hunk 1]
```diff
@@ -1,101 +1,5 @@
...
-- include: juno-cluster-resources.yml
-  when: openstack_release == 'juno'
-  tags: cinder
-
-- include: cluster-constraints.yml
-  tags: cinder
 
+- name: run cinder role based on juno release
+  include: juno.yml
+  when openstack_release == 'juno'
```
Comment (mtnbikenc) [inline roles/cinder/tasks/main.yml]: Needs a colon after `when`

File: roles/cinder/vars/main.yml [hunk 2]
```diff
@@ -3,3 +3,4 @@ memcache_list: "{{ memcached_servers | join(', ') }}"
 public_ipaddr: "{{ hostvars[inventory_hostname][ 'ansible_' + primary_if ].ipv4.address }}"
 internal_ipaddr: "{{ hostvars[inventory_hostname][ 'ansible_' + private_if ].ipv4.address }}"
 cinder_osapi_volume_listen: "{{ internal_ipaddr }}"
+rabbit_hosts: "{% for node in groups['controller'] %}{{ node }}:5672{% if not loop.last %},{% endif %}{% endfor %}"
```
Comment (mtnbikenc) [inline roles/cinder/vars/main.yml]: Are you wanting the name of the node here or the IP address?  Something like `{{ hostvars[node][ 'ansible_' + primary_if ].ipv4.address }}:5672` maybe in place of `{{ node }}:5672`?

**Comment by mtnbikenc** [comment]
Code review :+1:

**Comment by jcpowermac** [comment]
Code review :+1:

**Pull request merged**